monkey/agent_plugins/exploiters/mssql/config-schema.json

```diff
@@ -16,7 +16,7 @@
         },
         "try_discovered_mssql_ports": {
             "title": "Try discovered MSSQL ports",
-            "description": "Attempt to brute-force MSSQL on all TCP ports discovered from network scanning.",
+            "description": "Attempt the exploiter on all TCP ports discovered from network scanning which have MSSQL running. Ports with unknown services will also be included.",
```

Comment:
I think we should break this option up. 1 option for trying any known MSSQL ports. A separate option for trying UNKNOWN ports.